packages/wxt/src/__tests__/storage.test.ts

```diff
@@ -384,6 +384,56 @@ describe('Storage Utils', () => {
 
           expect(cb).not.toBeCalled();
         });
+
+        it('should watch multiple keys when passed an object', async () => {
+          const cb1 = vi.fn();
+          const cb2 = vi.fn();
+          const watchers = {
+            'local:key1': cb1,
+            'session:key2': cb2,
+          };
+
+          const unwatchers = storage.watch(watchers);
+
+          await storage.setItem('local:key1', 'value1');
+          await storage.setItem('session:key2', 'value2');
+
+          expect(cb1).toBeCalledTimes(1);
+          expect(cb1).toBeCalledWith('value1', null);
+          expect(cb2).toBeCalledTimes(1);
+          expect(cb2).toBeCalledWith('value2', null);
+
+          await storage.setItem('local:key1', 'new value1');
+          expect(cb1).toBeCalledTimes(2);
+          expect(cb2).toBeCalledTimes(1); // Still 1, not called again
+
+          // Test unwatching
+          Object.values(unwatchers).forEach((unwatch) => unwatch());
+          await storage.setItem('local:key1', 'newer value1');
+          await storage.setItem('session:key2', 'new value2');
+
+          expect(cb1).toBeCalledTimes(2); // Still 2, not called again
+          expect(cb2).toBeCalledTimes(1); // Still 1, not called again
+
+          expectTypeOf(unwatchers).toEqualTypeOf<{
+            'local:key1': Unwatch;
+            'session:key2': Unwatch;
+          }>();
```

Comment:
If you're watching a group of keys in a single call, we should return a single `unwatch` function to remove the listener for all of them at once. If someone needs to manage watches individually, they should just call watch with a single item at a time.

```suggestion
          expectTypeOf(unwatchers).toEqualTypeOf<Unwatch>();
```

Reply:
Yeah that makes more sense to be fair, shall I change it or will you in your commit?